packages/mongoose/src/config.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * © Copyright 2020 HP Development Company, L.P.
+ * SPDX-License-Identifier: MIT
+ */
+
+export default {
+    ALLOW_ESTIMATED_DOCUMENT_COUNT: process.env.ALLOW_ESTIMATED_DOCUMENT_COUNT || false
```

Comment:
I have doubts about passing the paremeter in this way or to pass an optional object with options to the createMongooseController instead (this way is easier to configure by deployment but the other way is more flexible)